app/models/transcribe.js

```diff
@@ -0,0 +1,38 @@
+`use strict`
+
+const Joi = require('joi')
+const MongoModels = require('mongo-models')
+
+const schema = Joi.object({
+    _id: Joi.object(),
+    path: Joi.string(),
+    subject: Joi.string().required(),
+    description: Joi.string().required(),
+    component: Joi.object(),
+    userId: Joi.string(),
+})
+
+class Transcribe extends MongoModels {
+    static create(obj) {
+        return new Prommise(async, (ok, ko) => {
```

Comment:
JS Promises should be spelled with one m. async followed by a comma seems to be a syntax error. ko and ok are parameters but are vague.

Reply:
Is it possible you are looking at an older commit, this file was deleted 2 weeks ago?